client/src/components/pages/Help.jsx

```diff
@@ -93,20 +116,33 @@ const Help = ({ authenticated }) => (
                     <div className="col-sm-10">
                         <h3 className="section-head">Export Data</h3>
                         <p className="pt-3">
-                            If you would like to export a CSV of your search results, submit a query in the <Link to="/database">database page</Link>, then click the "Export" button.
+                            If you would like to export a CSV of your search results, submit a query 
+                            in the <Link to="/database">database page</Link>, then click the "Export" 
+                            button.
                         </p>
-                        <center>
-                            <img src={require("../../images/help/Export/export_page.png")} alt="export page" />
-                        </center>
                         <p>
-                            You will be redirected to the "Export Data" page. Verify that the listed data is what you wish to export and delete any unwanted entries. You can choose to export this data separated by the analysis techniques used for each meteorite (this can result in multiple rows for a single meteorite that has multiple analysis techniques), or you can choose to discard the analysis techniques and export each meteorite in a single row of the CSV.
+                            <center>
+                                <img src={require("../../images/help/Export/export_page.png")} 
+                                    alt="export page" 
+                                />
+                            </center>
+                        </p>
+                        <p>
+                            You will be redirected to the "Export Data" page. Verify that the listed 
+                            data is what you wish to export and delete any unwanted entries. You can 
+                            choose to export this data separated by the analysis techniques used for 
+                            each meteorite (this can result in multiple rows for a single meteorite 
+                            that has multiple analysis techniques), or you can choose to discard the 
+                            analysis techniques and export each meteorite in a single row of the CSV.
                         </p>
```

Comment:
Do you know if some of the new line breaks in these HTML files impact the rendered website?